src/components/button.component.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const Button = (props) => (
+    <div className='button-container'>
+        <div className='button-group'>{props.buttonGroup.map(button => (
+            <button type='button' key={button.toString()}>{button}</button>
+            ))}
+        </div>
+    </div>
```

Comment:
- This component should accept the `button name` as an incoming prop  [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/acceptProp.png) which should be of type `String` [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png)